### DISCUSSION TITLE
Issue/PR in awaiting response

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle#29] [Show and tell]
Hey @kevinpapst,

With the labels that I have implemented, 
I don't know if you use it or not,
but every Issue/PR with label `Status: Needs Review` is awaiting of a response by you (or I forget to change it).

You can find them easily by using the filter `is:open label:"Status: Needs Review"` 
![image](https://user-images.githubusercontent.com/25293190/146407355-010c81a5-5e14-44c0-b7c7-7ef3cfba8886.png)

Or the link: https://github.com/kevinpapst/TablerBundle/labels/Status%3A%20Needs%20Review (that I have in my browser fav 😉 )

**Comment by kevinpapst** [comment]
Okay, I try to remember to remove the label after answering

**Reply by cavasinf** [reply]
I don't now what is the better.

1. If you remove the label every time, the history will be pretty messy.
2. If I didn't treat it or forget to remove it, you will still able to see it as "TODO".

2 seems to be a good way for today because we are only two on it.
But when it will be more "open", 1 is better.

**Reply by kevinpapst** [reply]
I haven't used labels in that way. I rarely change labels in issues. I am fine with every workflow.
I thought "Needs review" is more for PRs and "Discussion" or "RFC" is for issues

**Reply by cavasinf** [reply]
ATM I'll remove the label my self, and change it to "Status: Reviewed" when we are all good on the issue/PR.

**Reply by kevinpapst** [reply]
Alright sounds like a good plan 👍
